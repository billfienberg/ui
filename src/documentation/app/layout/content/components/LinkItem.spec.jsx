/* eslint-disable react/jsx-filename-extension */
import snap from 'jest-auto-snapshots';
import 'jest-styled-components';

import LinkItem from './LinkItem';

snap(LinkItem, './LinkItem.jsx');