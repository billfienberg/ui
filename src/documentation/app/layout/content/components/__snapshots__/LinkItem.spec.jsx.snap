// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-auto-snapshots > LinkItem Matches snapshot when passed only required props 1`] = `
<ForwardRef(Styled(Link))
  to="/jest-auto-snapshots String Fixture"
>
  <ForwardRef(styled.div)>
    <ForwardRef(styled.h3)>
      â†’ jest-auto-snapshots String Fixture
    </ForwardRef(styled.h3)>
  </ForwardRef(styled.div)>
</ForwardRef(Styled(Link))>
`;
